test/k8sT/Updates.go

```diff
@@ -248,7 +263,7 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldHelmChartVers
 		cleanupCiliumState(filepath.Join(kubectl.BasePath(), helpers.HelmTemplate), newHelmChartVersion, "", newImageVersion, "")
 
 		By("Cleaning Cilium state (%s)", oldImageVersion)
-		cleanupCiliumState("cilium/cilium", oldHelmChartVersion, "quay.io/cilium/cilium", oldImageVersion, "")
+		cleanupCiliumState(stableChartPath, oldHelmChartVersion, "quay.io/cilium/cilium", oldImageVersion, "")
```

Comment:
@aanm https://github.com/cilium/cilium/pull/19710 patch is against `test/k8s/updates.go` which I guess is `test/k8sT/Updates.go`  (this file) in v1.9.

I have kept here `"quay.io/cilium/cilium"` as `imageName` from the v1.9 branch (it's `"quay.io/cilium/cilium-ci"` in the upstream PR), please advise if this need to be updated as well.

Reply:
this sounds good

Reply:
Thanks for confirming, will do the same for v1.10/v1.11 if applicable.

Reply:
This should indeed be `-ci`